packages/core/src/plugins/plugin.ts

```diff
@@ -1,3 +1,5 @@
+import type { Runtime } from "../index.ts";
```

Comment:
weird import, i don't think importing from index.ts is right. shouldn't you import from ../runtime/runtime.ts

Reply:
Oopsy

Reply:
Fixed, thank you.